src/utils/getSortedGoods.ts

```diff
@@ -0,0 +1,36 @@
+import { SortType } from '../types/SortType';
+
+interface SortingFilters {
+  sortField: SortType;
+  isReversed: boolean;
+}
+
+export function getSortedGoods(
+  goods: string[],
+  { sortField, isReversed }: SortingFilters,
+) {
+  let preparedGoods = [...goods];
+
+  if (sortField) {
+    preparedGoods = preparedGoods.sort(
+      (currentGood: string, nextGood: string) => {
+        switch (sortField) {
+          case SortType.ALPHABETIBALLY:
```

Comment:
There's a typo in the `SortType` value. It should be `ALPHABETICALLY` instead of `ALPHABETIBALLY`. This will cause a bug since the string won't match the expected enum value, and the sorting won't work as intended.